x-pack/test/functional/apps/maps/group4/discover.js

```diff
@@ -42,10 +45,7 @@ export default function ({ getService, getPageObjects }) {
 
     it('should link geo_point fields to Maps application with time and query context', async () => {
       await PageObjects.discover.selectIndexPattern('logstash-*');
-      await PageObjects.timePicker.setAbsoluteRange(
-        'Sep 22, 2015 @ 00:00:00.000',
-        'Sep 22, 2015 @ 04:00:00.000'
-      );
+      await PageObjects.common.setTime({ from, to });
```

Comment:
You generally have to do this before navigating to the app.

Reply:
Yep that was the problem. Fixes test failures.